src/templates/base.html.j2

```diff
@@ -8,6 +8,9 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.min.css" 
       integrity="sha256-eSi1q2PG6J7g7ib17yAaWMcrr5GrtohYChqibrV7PBE="
       crossorigin="anonymous" />
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"
```

Comment:
What's the purpose of this change? If want_js is false, then javascript shouldn't get loaded...

Reply:
JQuery needs to be loaded before the script section so that `$` is defined.  I suppose this should be surrounded by the `want_js` if's.

Reply:
Actually it looks like you added the Freshdesk Support Widget to base.html.j2 which is all JavaScript, so at this point I guess you might as well get rid of `want_js` altogether...

Reply:
Removed want_js

Reply:
Can you edit contacts.html.j2 and generate_downtime_form.html.j2 to remove the `set want_js = true` lines?